operators/apimatic-codegen-operator/v1.8.3/metadata/annotations.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: apimatic-codegen-operator
+  operators.operatorframework.io.bundle.channels.v1: beta
```

Comment:
@Obaid-Rehman There is a missing attribute of `operators.operatorframework.io.bundle.channel.default.v1` that needs to be added to this file.

Reply:
Thanks @acornett21 :)....will get this fixed.